### PR TITLE
Deploy kubernetes external secrets controller and CRD in k8s-prow-bui…

### PR DESCRIPTION
**Pull request by chaodaiG**
…lds cluster

### CODE REVIEW

File: config/prow/cluster/build_kubernetes-external-secrets_crd.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+./kubernetes-external-secrets_crd.yaml
```
Comment (fejta) [inline config/prow/cluster/build_kubernetes-external-secrets_crd.yaml:1]: Shouldn't this be a yaml file?

Comment (chaodaiG) [reply config/prow/cluster/build_kubernetes-external-secrets_crd.yaml:1]: This is a symlink. I meant to use the same file, but this rule:
```
 component(
        "kubernetes-external-secrets",
        "deployment",
        "rbac",
        "service",
        "crd",
        cluster = BUILD_CLUSTER,
    ),
```
kept on complaining the rule `kubernetes-external-secrets-deployment` already exists, and creating this symlink seems solving the problem

**Review by fejta — APPROVED** [review]

**Pull request merged**